sktime/forecasting/pytorchforecasting.py

```diff
@@ -378,3 +523,529 @@ def get_test_params(cls, parameter_set="default"):
+        """
+        try:
+            _check_soft_dependencies("pytorch_forecasting", severity="error")
+        except ModuleNotFoundError:
+            params = [
+                {
+                    "trainer_params": {
+                        "max_epochs": 1,  # for quick test
+                        "limit_train_batches": 10,  # for quick test
+                    },
+                    "dataset_params": {
+                        "max_encoder_length": 3,
+                    },
+                    "random_log_path": True,  # fix multiprocess file access error in CI
+                    "deterministic": True,  # to pass test_score
+                },
+                {
+                    "trainer_params": {
+                        "max_epochs": 1,  # for quick test
+                        "limit_train_batches": 10,  # for quick test
+                    },
+                    "model_params": {
+                        "cell_type": "GRU",
+                        "rnn_layers": 3,
+                    },
+                    "dataset_params": {
+                        "max_encoder_length": 3,
+                    },
+                    "random_log_path": True,  # fix multiprocess file access error in CI
+                    "deterministic": True,  # to pass test_score
+                },
+            ]
+        else:
+            from lightning.pytorch.callbacks import EarlyStopping
+
+            early_stop_callback = EarlyStopping(
+                monitor="val_loss",
+                min_delta=1e-2,
+                patience=3,
+                verbose=False,
+                mode="min",
+            )
+            params = [
+                {
+                    "trainer_params": {
+                        "max_epochs": 1,  # for quick test
+                        "limit_train_batches": 10,  # for quick test
+                    },
+                    "dataset_params": {
+                        "max_encoder_length": 3,
+                    },
+                    "random_log_path": True,  # fix multiprocess file access error in CI
+                    "deterministic": True,  # to pass test_score
+                },
+                {
+                    "trainer_params": {
+                        "callbacks": [early_stop_callback],
+                        "max_epochs": 1,  # for quick test
+                        "limit_train_batches": 10,  # for quick test
+                    },
+                    "model_params": {
+                        "cell_type": "GRU",
+                        "rnn_layers": 3,
+                    },
+                    "dataset_params": {
+                        "max_encoder_length": 3,
+                    },
+                    "random_log_path": True,  # fix multiprocess file access error in CI
+                    "deterministic": True,  # to pass test_score
+                },
+            ]
+
+        return params
+
+
+class PytorchForecastingNHiTS(_PytorchForecastingAdapter):
```

Comment:
Is it already part of an other PR?

Reply:
Yes, DeepAR and NHiTS are part of https://github.com/sktime/sktime/pull/6551, this branch is inherited from that.